OasisDEX/oasis-borrow
# Institutional Vaults view.

# [Create left side of manage institutional vault page](https://app.shortcut.com/oazo-apps/story/3570/create-left-side-of-manage-institutional-vault-page-inside-storybook)

Plus this story: [Create new form for institutional vaults based on borrow UI](https://app.shortcut.com/oazo-apps/story/3548/create-new-form-for-institutional-vaults-based-on-borrow-ui)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- Add ManageInstiVaultContainer.
- Refactor some components of ManageVaultContainer (borrow), so they can be customized a bit, enough to accommodate for institutional vaults.
- Use ManageInstiVaultContainer on GeneralManageVaultView, with mock data.

## How to test 🧪
  <Please explain how to test your changes>

- On vaultType.ts file: Add your borrow vault id to `isInstiVault` function, then your vault would show up as an institutional vault. Everything should work the same as borrow.
- The normal borrow vaults should still work normally as well (considering it's been refactored).
- See design here: https://www.notion.so/oazo/Institutional-vaults-DSS-charter-0e1ad01cac574f4cbcdbbeec8d18515d
